src/System.Private.ServiceModel/tests/Scenarios/Contract/XmlSerializer/XmlSerializerFormatSoapTest.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.ServiceModel;
+using System.ServiceModel.Channels;
+using System.Threading.Tasks;
+using Infrastructure.Common;
+using Xunit;
+
+public static partial class XmlSerializerFormatTests
```

Comment:
I know this Is nit-picky but we would like to have a consistent 'feel' across all the tests, look at some of the other tests for examples but could you...
- Add some comment blocks for sections of the tests like...
             // *** SETUP *** \\
            // *** EXECUTE *** \\
           // *** CLEANUP *** \\
- Use try/catch/finally in these tests it doesn't look like you need the 'Catch' but in the finally we make sure we cleanup by calling `ScenarioTestHelpers.CloseCommunicationObjects`